src/Form.js

```diff
@@ -23,6 +24,11 @@ class Form {
         for (const field in data) {
             this[field] = data[field];
         }
+
+        this.__options = { 
```

Comment:
Would be breaking if you have a form that uses options. Also don't want to pollute the public properties of this class since they could always collide with form fields.

Since `options` is only for private use, it doesn't need to be pretty. An underscore or double underscore is a pretty conventional way to mark a property private in JavaScript (pretty much all major libraries do this in some place), so I don't really see anything wrong with it.